Gopkg.lock

```diff
@@ -236,12 +236,6 @@
   revision = "8bbc2356beaf021b04c9bd5cdc76ea5a7ccb40ec"
   version = "v1.3.0"
 
-[[projects]]
-  branch = "master"
-  name = "github.com/fullsailor/pkcs7"
-  packages = ["."]
-  revision = "a009d8d7de53d9503c797cb8ec66fa3b21eed209"
-
```

Comment:
No idea why this got removed.

Reply:
Probably no longer used, that's great ! Deps--